tasks/fix-cat2.yml

```diff
@@ -1089,15 +1089,31 @@
       - RHEL-07-021300
 
 - name: "MEDIUM | RHEL-07-021620 | PATCH | The file integrity tool must use FIPS 140-2 approved cryptographic hashes for validating file contents and directories."
-  command: "true"
-  changed_when: no
+  block:
+    - name: "Replace sha256+sha512 entries with sha512"
+      replace:
+        path: /etc/aide.conf
+        regexp: '([A-Z]+ = .*)(sha256\+sha512)(.*)'
+        replace: '\1sha512\3'
+      register: rhel_07_021620_sha256_512_result
+      failed_when:
+        - rhel_07_021620_sha256_512_result | failed
+        - rhel_07_021620_sha256_512_result.rc != 257
+    - name: "Replace sha256 entries with sha512"
+      replace:
+        path: aide.conf
```

Comment:
This should be `/etc/aide.conf` I think?

Reply:
Yep. Bad test environment. Thought I had all those fixed. Will adjust.